### PR TITLE
Pw dev 2

### PR DESCRIPTION
**Pull request by piercedwilson**
Created calendar.html and added basic navbar

### CODE REVIEW

File: capstone/fitness/src/main/webapp/calendar.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <script src="script.js"></script>
+  </head>
+  <!-- basic navbar for navigation during development. can be stylized later. -->
+  <div class="topnav">
+    <a class="active" href="/">Home</a>
+    <a href="/calendar.html">Calendar</a>
+    <a href="/">Progress</a>
+    <a href="/">Data</a>
+  </div>
+  <body onload="getCalendarInfo();">
```
Comment (lupengscu) [inline capstone/fitness/src/main/webapp/calendar.html:18]: Can you add a simple implementation of getCalendarInfo() that gets data from CalendarServlet?

**Review by lupengscu — CHANGES_REQUESTED** [review]

File: capstone/fitness/src/main/webapp/calendar.html (hunk 1 above)
Comment (piercedwilson) [reply capstone/fitness/src/main/webapp/calendar.html:18]: Yes

File: capstone/fitness/src/main/webapp/calendar.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<!-- HTML for calendar page. Containers load in information from java using the onload in body. -->
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Your Calendar</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <!-- basic navbar for navigation during development. can be stylized later. -->
```
Comment (sliang3) [inline capstone/fitness/src/main/webapp/calendar.html]: nit: Please format comments like sentences (capitalization).

Also for comments like "can be stylized later", would recommend putting a new line and adding "TODO(piercedwilson): " in front so you remember to get back to it later.

File: capstone/fitness/src/main/webapp/calendar.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    <a href="/calendar.html">Calendar</a>
+    <a href="/">Progress</a>
+    <a href="/">Data</a>
+  </div>
+  <body onload="getCalendarInfo();">
+    <div id="content">
+    <br><br>
+    <h1 style="background-color:Tomato;">Your Calendar</h1>
+    <!-- p1 loads in user's nickname and a short message in Java -->
+    <p id = "p1">Hi [User]!...</p>
```
Comment (sliang3) [inline capstone/fitness/src/main/webapp/calendar.html]: Would recommend using a more descriptive name here than p1.

**Review by sliang3 — APPROVED** [review]

**Review by GabrielGonzalezDiaz — APPROVED** [review]

File: capstone/fitness/src/main/java/com/google/sps/servlets/CalendarServlet.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.security.GeneralSecurityException;
+import java.util.Collections;
+import java.util.List;
+import com.google.gson.Gson;
+import java.util.Date;
+
+/** Servlet that will deal with calendar. */
+@WebServlet("/calendar-servlet")
```
Comment (lupengscu) [inline capstone/fitness/src/main/java/com/google/sps/servlets/CalendarServlet.java:34]: Remove -servlet in the path

**Review by lupengscu — APPROVED** [review]

File: capstone/fitness/src/main/java/com/google/sps/servlets/CalendarServlet.java (hunk 4 above)
Comment (piercedwilson) [reply capstone/fitness/src/main/java/com/google/sps/servlets/CalendarServlet.java:34]: Okay.

**Pull request merged**